src/descriptor_runner/descriptor_runner/descriptor_runner_fallback.ts

```diff
@@ -111,10 +113,35 @@ export default class DescriptorRunnerFallback extends DescriptorRunner<GraphDesc
     private async compile(): Promise<void> {
         if (!this.descriptor) throw new Error('Descriptor is not loaded');
 
-        let dnn_fallback_kernel: any = null;
-        eval(this.descriptor.kernel_source);
+        let local_dnn_fallback_kernel = null;
+
+        function loadScript(url){
+            var script = <any>document.createElement("script")
+            script.type = "text/javascript";
+            let promise = new Promise((resolve, reject)=>{
+               if (script.readyState){  //IE
+                    script.onreadystatechange = function(){
+                        if (script.readyState == "loaded" ||
+                                script.readyState == "complete"){
+                            script.onreadystatechange = null;
+                            resolve();
+                        }
+                    };
+                } else {  //Others
+                    script.onload = function(){
+                        resolve();
+                    };
+                }
+            })
+
+            script.src = url;
+            document.getElementsByTagName("head")[0].appendChild(script);
+            return promise
+        }
 
-        this.kernelObj = dnn_fallback_kernel;
+        await loadScript(`${this.directory}/kernels_fallback.js`);
+        local_dnn_fallback_kernel = (<any>window).dnn_fallback_kernel
```

Comment:
use `window as any` instead of `<any>`

Reply:
Please add comment `// "window.dnn_fallback_kernel" is defined in "kernels_fallback.js"`.